proto/streamlit/proto/Common.proto

```diff
@@ -79,7 +79,8 @@ message UploadedFileInfo {
   string name = 2;
 
   // The size of this file in bytes.
-  uint32 size = 3;
+  // We changed the type from uint32 to fix https://github.com/streamlit/streamlit/issues/5938. Based on the protobuf docs (https://protobuf.dev/programming-guides/proto3/), this is a compatible change.
```

Comment:
Hm, given that this change is a legal one, I feel like we don't really need a comment here explaining that we did something/why.